tcp_packet_conn.go

```diff
@@ -0,0 +1,232 @@
+package ice
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"sync"
+	"time"
+
+	"github.com/pion/logging"
+)
+
+type tcpPacketConn struct {
+	params *tcpPacketParams
+
+	// conns is a map of net.Conns indexed by remote net.Addr.String()
+	conns map[string]net.Conn
+
+	recvChan chan streamingPacket
+
+	mu         sync.Mutex
+	wg         sync.WaitGroup
+	closedChan chan struct{}
+	closeOnce  sync.Once
+}
+
+var _ net.PacketConn = &tcpPacketConn{}
```

Comment:
Is this just to assert that it satisfies an interface?

Reply:
Yes

Reply:
It should be ok to drop this (I think?) since it is all internal. We don't have to worry about accidentally breaking intreface. Or maybe if you want to keep I would put this in a `_test.go` 

We do this in RTCP, but I think it is kind of confusing.

Reply:
Removed!